app/src/main/java/org/gem/indo/dooit/views/onboarding/OnboardingWebActivity.java

```diff
@@ -6,6 +6,8 @@
 import android.net.NetworkInfo;
```

Comment:
This was my doing. Will be fixed in https://github.com/praekeltfoundation/gem-bbb-indo/pull/163